model/SharedStimulusPackageImporter.php

```diff
@@ -179,10 +180,17 @@ private function getSharedStimulusStylesheets(string $extractPath): array
         return $cssFileInfoArray;
     }
 
-    public function isFileExtension(SplFileInfo $file, string $extension): bool
+    private function isFileExtension(SplFileInfo $file, string $extension): bool
     {
         if ($file->isFile()) {
```

Comment:
In this case, an early return would make the code less nested

```suggestion
        if (!$file->isFile()) {
            return false;
```